sql/src/main/java/io/crate/operation/NodeOperation.java

```diff
@@ -75,6 +85,7 @@ public int downstreamExecutionPhaseId() {
     public void readFrom(StreamInput in) throws IOException {
         executionPhase = ExecutionPhases.fromStream(in);
         downstreamExecutionPhaseId = in.readVInt();
+        downstreamExecutionPhaseInputId = in.readByte();
```

Comment:
not sure if we want to have CHANGES conflicts each time we rebase cross-joins against master. I'll add a task so that we don't forget it.